README.md

```diff
@@ -16,7 +16,7 @@ returned object of `new WebSocket()` calls with the socket you passed in.
 
 ```javascript
 const socket = new Fauxnix(function() {
-  this.receive("topic", "event", function() {
+  this.receive("topic", "event", function(payload) {
     return { status: "ok", response: { id: 1 } };
```

Comment:
Yeah that would be fine. I just wouldn't want it to confuse someone by there being this option and not being used or knowing what it is.